evm-contracts/test/v0.4/SignedSafeMath.test.ts

```diff
@@ -15,7 +15,7 @@ beforeAll(async () => {
 describe('SignedSafeMath', () => {
   // a version of the adder contract where we make all ABI exposed functions constant
   // TODO: submit upstream PR to support constant contract type generation
-  let adder: contract.Instance<ConcreteSignedSafeMathFactory>
+  let cssm: contract.Instance<ConcreteSignedSafeMathFactory>
```

Comment:
Now that this is officially in the OZ contracts, as opposed to previously beta, it seems like it can be treated as vendored and we don't need to test it. Maybe we can remove these tests in a separate PR, but in general I'd like to avoid us testing vendored code.

Reply:
Always good with deleting, no need to wait.